pcdsdevices/attenuator.py

```diff
@@ -27,6 +28,31 @@
 logger = logging.getLogger(__name__)
 MAX_FILTERS = 12
 
+class Gatt_Y_aperture(TwinCATStatePMPS):
```

Comment:
Trying to avoid the `_` in the name: maybe `GattApertureX` and `GattApertureY`?

Reply:
https://peps.python.org/pep-0008/#class-names